scipy/signal/tests/test_filter_design.py

```diff
@@ -1189,6 +1400,24 @@ def test_analog(self):
 
         assert_equal(buttord(1, 1.2, 1, 80, analog=True)[0], 55)
 
+    def test_fs_param(self):
+        wp = [4410, 11025]
+        ws = [2205, 13230]
+        rp = 3
+        rs = 80
+        fs = 44100
+        N, Wn = buttord(wp, ws, rp, rs, False, fs=fs)
+        b, a = butter(N, Wn, 'bandpass', False, fs=fs)
+        w, h = freqz(b, a, fs=fs)
+        assert_array_less(-rp - 0.1,
+                          dB(h[np.logical_and(wp[0] <= w, w <= wp[1])]))
+        assert_array_less(dB(h[np.logical_or(w <= ws[0], ws[1] <= w)]),
+                          -rs + 0.1)
+
+        assert_equal(N, 18)
+        assert_allclose(Wn, [4409.722701715714, 11025.47178084662],
```

Comment:
How did you derive such precise values for the expected results?  Add a comment explaining where these numbers came from, so future maintainers can reproduce those expected values.

Reply:
I think the hardcoded values were to detect changes in the behavior of whether the filter "hugs" the inner or outer boundary.  

`buttord`'s hugging behavior was changed in https://github.com/scipy/scipy/pull/3235#issuecomment-33882609 

Discussion was here: https://github.com/scipy/scipy/issues/3219#issuecomment-32824200 

some discussion of 0.1 dB tolerances in both of those

Reply:
Anyway can those test changes be a separate PR, since they're about the `...ord` functions and not about the fs parameter?